code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -51,6 +51,7 @@ GLOBAL_LIST_INIT(metal_recipes, list ( \
 	new/datum/stack_recipe("closet", /obj/structure/closet, 2, time = 15, one_per_turf = TRUE, on_floor = TRUE), \
 	null, \
 	new/datum/stack_recipe("canister", /obj/machinery/portable_atmospherics/canister, 10, time = 15, one_per_turf = TRUE, on_floor = TRUE), \
+	new/datum/stack_recipe("trash bin", /obj/structure/closet/crate/bin, 10, time = 15, one_per_turf = TRUE, on_floor = TRUE),\
```

Comment:
Blacklist them from the cargo export as trilby said.
https://github.com/Citadel-Station-13/Citadel-Station-13/blob/f7d8be598947c90fd8156ff683d36a09742ef31d/code/modules/cargo/exports/large_objects.dm#L9

Reply:
This has already been done.